src/api/spec/features/webui/login_spec.rb

```diff
@@ -91,4 +92,84 @@
     expect(page).not_to have_css("a#link-to-user-home")
     expect(page).to have_link("Log")
   end
+
+  context 'in kerberos mode' do
+    before do
+      stub_const('CONFIG', CONFIG.merge({
+        'kerberos_service_principal' => 'HTTP/obs.test.com@test_realm.com',
+        'kerberos_realm'             => 'test_realm.com',
+        'kerberos_mode'              => true
+      }))
+    end
+
+    context 'for a normal request that requires login' do
```

Comment:
"normal request" here is pretty vague, maybe change this line to something like this?
```ruby
context 'a request without the authorization header' do
```